resources/views/daily_menus/show.blade.php

```diff
@@ -0,0 +1,63 @@
+@extends('layouts.master')
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">{{ __('Daily Menu For') }}<span class="label label-info" id="$menuDate">{{ $date }}</span></h3>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div class="dataTables_wrapper form-inline dt-bootstrap">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table class="table table-bordered table-striped dataTable table-hover" role="grid">
+                            <thead>
+                            <tr role="row">
+                                <th style="width: 1em">{{ __('ID') }}</th>
+                                <th style="width: 6em">{{ __('Food Name') }}</th>
+                                <th style="width: 6em">{{ __('Category Name') }}</th>
+                                <th style="width: 2em">{{ __('Price') }}</th>
+                                <th style="width: 4em">{{ __('Created At') }}</th>
+                                <th style="width: 4em">{{ __('Updated At') }}</th>
+                                <th style="width: 1em">{{ __('Quantity') }}</th>
+                                <th style="width: 2em">{{ __('Actions') }}</th>
+                            </tr>
+                            </thead>
+                            <tbody id="bodyTable">
+                            @foreach ($menuOnDate as $menuItem)
```

Comment:
I think its dont have case menuOnDate null because in list dailymenu just show list date was has Food in its.
When we delete item in menu, if it has nothing then auto redirect into list menu
But i have updated {{!empty($menuOnDate)}} case